src/metadata.json

```diff
@@ -6,7 +6,7 @@
     "williambrendaw@icloud.com"
   ],
   "shell-version": [
-    "43"
+    "44"
```

Comment:
This makes a soft version fork.  If no other code changes were needed and the new version just worked, you probably should have kept both 43 and 44 in the array here.  Why it matters: if some defect/bug is found (unrelated to GNOME version), and you want to do a fix, you essentially have to make two fixes (one each for GNOME 43 & 44 users, separately), and then release two new extension versions on the Extensions site.  The reason there was only 43 here and not 42 and below was that there was a significant refactor done to bring up the code for GNOME 43, so earlier versions of GNOME weren't tested at that time. (I was the author of the GNOME 43 branch.)  See https://github.com/brendaw/add-username-toppanel/pull/11/commits/93ab0865103cd296446f35acd0716482d8b242e0

Reply:
That's really true, Josh, and I totally overlooked it. 

Thanks for this heads-up I fixed on main to have this two versions and the now active at GNOME Extensions have this line of versions.

Thanks again for your amazing help on our extension.